src/Tribe/Views/V2/Customizer.php

```diff
@@ -23,12 +23,381 @@ class Customizer {
+					color: ' . $accent_color_active . ';
+				}
+			';
+
+			$css_template .= '
+				.tribe-theme-twentyseventeen .tribe-events .tribe-events-calendar-month__day-cell--selected:hover,
+				.tribe-theme-twentyseventeen .tribe-events .tribe-events-calendar-month__day-cell--selected:focus {
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			$css_template .= '
+				.tribe-theme-twentytwenty .tribe-events .tribe-events-calendar-month__day-cell--selected {
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			// overrides for tec views/full/month/_mobile-events.pcss.
+			$css_template .= '
+				.tribe-events .tribe-events-calendar-month-mobile-events__mobile-event-datetime-featured-text {
+					color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			// overrides for tec views/full/month/_multiday-events.pcss.
+			$css_template .= '
+				.tribe-events .tribe-events-calendar-month__multiday-event-bar-inner {
+					background-color: ' . $accent_color_multiday . ';
+				}
+			';
+
+			$css_template .= '
+				.tribe-events .tribe-events-calendar-month__multiday-event-bar-inner--hover,
+				.tribe-events .tribe-events-calendar-month__multiday-event-bar-inner--focus {
+					background-color: ' . $accent_color_multiday_hover . ';
+				}
+			';
+
+			// overrides for tec views/full/day/_event.pcss.
+			$css_template .= '
+				.tribe-events .tribe-events-calendar-day__event--featured:after {
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			$css_template .= '
+				.tribe-common--breakpoint-medium.tribe-events .tribe-events-calendar-day__event-datetime-featured-text {
+					color: <%= global_elements.accent_color %>;
+				}
+			';
+		}
+
+		if ( $customizer->has_option( $section->ID, 'link_color' ) ) {
+			$css_template .= '
+				.tribe-events-single-event-description a,
+				.tribe-events-event-url a,
+				.tribe-venue-url a,
+				.tribe-organizer-url a,
+
+				.tribe-events-pro .tribe-events-pro-organizer__meta-website a,
+				.tribe-block__organizer__website a,
+				.tribe-events-pro .tribe-events-pro-venue__meta-website a,
+				.tribe-block__venue__website a,
+				.tribe_events p a {
+					color: <%= global_elements.link_color %>;
+				}
+			';
+		}
+
+		return $css_template;
+	}
+
+	/**
+	 * Filters the Single Event section CSS template to add Views v2 related style templates to it.
+	 *
+	 * @since TBD
+	 *
+	 * @param string                     $css_template The CSS template, as produced by the Global Elements.
+	 * @param Tribe__Customizer__Section $section      The Single Event section.
+	 * @param Tribe__Customizer          $customizer   The current Customizer instance.
+	 *
+	 * @return string The filtered CSS template.
+	 */
+	public function filter_single_event_css_template(  $css_template, $section, $customizer) {
```

Comment:
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 2 found
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: No space before closing parenthesis is prohibited

<sub>hash: 5a390ed3841dded39878e0f7a25aeecb</sub>